source/channels.yaml

```diff
@@ -410,6 +410,44 @@
               additionalProperties:
                 $ref: "#/definitions/ChannelMember"
 
+  '/teams/{team_id}/channels/{channel_id}/update_member_roles':
```

Comment:
Hmmm, ok yeah let's be consistent with the `/teams/{team_id}/update_member_roles` since that's the closest endpoint to this one